modules/bullet/space_bullet.cpp

```diff
@@ -592,7 +592,16 @@ void SpaceBullet::create_empty_world(bool p_create_soft_world) {
 
 	if (p_create_soft_world) {
 		dynamicsWorld = new (world_mem) btSoftRigidDynamicsWorld(dispatcher, broadphase, solver, collisionConfiguration);
+
+#if defined(_MSC_VER)
+#pragma warning(push)
+#pragma warning(disable : 4316)
+#endif
```

Comment:
I don't think we should have to hack around this, if MSVC doesn't like `bulletnew` we should likely fix that upstream in Bullet. CC @AndreaCatania

Reply:
I added the hack here for this one because `btSoftBodyWorldInfo` is coming from 3rd party code. I will look into changing `bulletnew`